tests/integration_tests/data_steward/utils/participant_summary_requests_test.py

```diff
@@ -0,0 +1,197 @@
+            'resource': {
+                'participantId': 'P222',
+                'suspensionStatus': 'NO_CONTACT',
+                'suspensionTime': '2018-12-07T08:21:14'
+            }
+        }]
+
+        self.json_response_entry = {
+            'entry': [{
+                'fullUrl':
+                    'https//foo_project.appspot.com/rdr/v1/Participant/P111/Summary',
+                'resource': {
+                    'participantId': 'P111',
+                    'suspensionStatus': 'NO_CONTACT',
+                    'suspensionTime': '2018-12-07T08:21:14'
+                }
+            }, {
+                'fullUrl':
+                    'https//foo_project.appspot.com/rdr/v1/Participant/P222/Summary',
+                'resource': {
+                    'participantId': 'P222',
+                    'suspensionStatus': 'NO_CONTACT',
+                    'suspensionTime': '2018-12-07T08:21:14'
+                }
+            }]
+        }
+
+        super().setUp()
+
+    def test_get_access_token(self):
+        # pre conditions
+        scopes = [
+            'https://www.googleapis.com/auth/cloud-platform', 'email', 'profile'
+        ]
+        credentials, _ = default()
+        credentials = auth.delegated_credentials(credentials, scopes=scopes)
+        request = req.Request()
+        credentials.refresh(request)
+        expected_access_token = credentials.token
+
+        # test
+        actual_access_token = psr.get_access_token()
+
+        # post conditions
+        self.assertEqual(actual_access_token, expected_access_token)
+
+    @mock.patch('utils.participant_summary_requests.requests.get')
+    def test_get_participant_data(self, mock_get):
+        # pre conditions
+        mock_get.return_value.status_code = 200
+        mock_get.return_value.json.return_value = self.json_response_entry
+
+        # test
+        expected_response = psr.get_participant_data(self.url, self.headers)
+
+        # post conditions
+        self.assertEqual(expected_response, self.participant_data)
+
+    def test_get_deactivated_participants_parameters(self):
+        # Parameter check tests
+        self.assertRaises(RuntimeError, psr.get_deactivated_participants, None,
+                          self.dataset_id, self.tablename, self.columns)
+        self.assertRaises(RuntimeError, psr.get_deactivated_participants,
+                          self.project_id, None, self.tablename, self.columns)
+        self.assertRaises(RuntimeError, psr.get_deactivated_participants,
+                          self.project_id, self.dataset_id, None, self.columns)
+        self.assertRaises(RuntimeError, psr.get_deactivated_participants,
+                          self.project_id, self.dataset_id, self.tablename,
+                          None)
+
+    @mock.patch('utils.participant_summary_requests.store_participant_data')
+    @mock.patch(
+        'utils.participant_summary_requests.get_deactivated_participants')
```

Comment:
This test can't be testing `get_deactivated_participants` or `store_participant_data` because both functions are mocked.  As part of the integration test, you want your tested code to actually build the table in BigQuery to make sure your test is verifying your code functions correctly.  The only part of the test you want to mock is the response from the RDR API, `@mock.patch('utils.participant_summary_requests.requests.get')`.  

I've been looking at how to do this and have a partial patch file.  We can talk about this via a call if you would like.